### ISSUE TITLE
[stable/prometheus-rabbitmq-exporter] doesn't show up in the targets

### ISSUE THREAD

**Issue by jackivanov** [helm/charts#15968]
**Describe the bug**
The rabbitmq exporter doesn't appear in the targets

<details><summary>Screenshot</summary><p>

![prometheus targets](https://i.imgur.com/h6d2U5W.png)
</p></details>

**Version of Helm and Kubernetes**:
```
$ helm version
Client: &version.Version{SemVer:"v2.14.2", GitCommit:"a8b13cc5ab6a7dbef0a58f5061bcc7c0c61598e7", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.2", GitCommit:"a8b13cc5ab6a7dbef0a58f5061bcc7c0c61598e7", GitTreeState:"clean"}
```

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.0", GitCommit:"e8462b5b5dc2584fdcd18e6bcfe9f1e4d970a529", GitTreeState:"clean", BuildDate:"2019-06-20T04:49:16Z", GoVersion:"go1.12.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.3", GitCommit:"5e53fd6bc17c0dec8434817e69b04a25d8ae0ff0", GitTreeState:"clean", BuildDate:"2019-06-06T01:36:19Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:
stable/prometheus-rabbitmq-exporter

**What happened**:
I've deployed prometheus from `stable/prometheus-operator`. Then I've deployed Elasticsearch and RabbitmQ exporters from `stable/elasticsearch-exporter` `stable/prometheus-rabbitmq-exporter` with the following config:

```yaml
elasticsearch-exporter:
  serviceMonitor:
    enabled: true
    interval: 30s
    labels:
      release: prometheus-operator
  es:
    uri: "http://es-development-master:9200"

prometheus-rabbitmq-exporter:
  rabbitmq:
    url: http://rabbitmq-development:15672
    user: admin
    password: FzWRcdWwmOtOM8b_o0
  prometheus:
    monitor:
      enabled: true
      interval: 30s
      additionalLabels:
        release: prometheus-operator
      namespace:
        - development
```

Elasticsearch appeared in the targets, but RabbitmQ didn't

**What you expected to happen**:

The rabbitmq exporter appears in the targets

**How to reproduce it** (as minimally and precisely as possible):

Install stable/prometheus-rabbitmq-exporter with the settings above or the default ones

**Anything else we need to know**:

**Comment by winromulus** [comment]
I also have the same issue with the rabbitmq exporter. It does not show up in the targets list

**Comment by RaymondArias** [comment]
What is the exporter logging?

**Comment by 1ucasun** [comment]
I guess that the chart doesn't support prometheus-operator servicemonitor sd

**Comment by winromulus** [comment]
Any news on this or any idea what needs to be fixed?

**Comment by RaymondArias** [comment]
#15801  should fix this issue whenever it is merged in

**Comment by RaymondArias** [comment]
Actually it may be fixed already with this https://github.com/helm/charts/commit/0a9b8a2db04baee6573f9d30b343f492e4fb1642

Try your rabbitmq exporter again with the latest chart

**Comment by maorfr** [comment]
closed by #15801

**Comment by geobos23** [comment]
Aug 13 - Issue was fixed by #0a9b8a2 (PR #16134) -  `0.5.2` 
Sep 15 - Issue happening again because PR #15801 - `0.5.3`
See more details at https://github.com/helm/charts/pull/15801#issuecomment-542181611
